components/content-blocks/Callout/CalloutEntry/index.js

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable react/prop-types */
+
+import PropTypes from "prop-types";
+import { useTranslation } from "react-i18next";
+import { makeDateString, useGlobalData } from "@/lib/utils";
+import { Text } from "../styles";
+import {
+  Section,
+  Inner,
+  SharePopup,
+  ImageWrapper,
+  ImageSticker,
+} from "./styles";
+import { Image } from "@rubin-epo/epo-react-lib";
+
+const getDateString = (newsDate, eventStart, eventEnd, lang) => {
+  if (newsDate) {
+    return makeDateString(newsDate, lang);
+  }
+
+  if (eventStart && eventEnd) {
+    return `${makeDateString(eventStart, lang)} - ${makeDateString(
+      eventEnd,
+      lang
+    )}`;
+  }
+
+  if (eventEnd) {
+    return makeDateString(eventEnd, lang);
+  }
+};
+
+export default function CalloutEntry({ callout }) {
```

Comment:
Found it was prudent for both Event and News callouts to be rendered by the same component, "Entry" felt then like the relevant term to include in the component name.